test/Interop/SwiftToCxx/structs/swift-struct-in-cxx.swift

```diff
@@ -1,20 +1,26 @@
 // RUN: %empty-directory(%t)
-// RUN: %target-swift-frontend %s -typecheck -module-name Structs -clang-header-expose-public-decls -emit-clang-header-path %t/structs.h
+// RUN: %target-swift-frontend %s -typecheck -emit-ir -module-name Structs -clang-header-expose-public-decls -emit-clang-header-path %t/structs.h
```

Comment:
Why does this need the IR now?

Reply:
Right now I'm getting the layout info about canonical SIL is constructed. I definitely need SIL to gather the layout info. That's what I mentioned in my description:

Note: the current header emission logic does not yet force SILGen/IRGen, so we will only emit types when the compiler gets to IRGen. This is something that will be addressed in the future (we should probably require SILGen/IRGen for header emission?)

I think that's something I should ask John in the sync-up as well.